php/class-photonfill.php

```diff
@@ -526,6 +581,33 @@ private function get_img_src( $attachment_id, $size = null, $default = false ) {
 		}
 
 		/**
+		 * Manipulate a external url img src
+		 * @param string $img_url
+		 * @param array. This should always be an array of breakpoint width and height
+		 * @param boolean. Should this be the default srcset for the img element.
+		 */
+		private function get_url_img_src( $img_url, $size = null, $default = false ) {
+			if ( ! empty( $img_url ) ) {
+				$width = $size[0];
+				$height = $size[1];
+
+				$img_src = array(
+					'width' => $width,
+					'height' => $height,
+					'default' => $default,
+				);
+
+				// Support jetpack photon and my_photon
+				$photon_url_function = photonfill_hook_prefix() . '_photon_url';
+				$img_src['url'] = $photon_url_function( $img_url, array( 'width' => $img_src['width'], 'height' => $img_src['height'] ) );
+				$img_src['url2x'] = $photon_url_function( $img_url, array( 'width' => absint( $img_src['width'] * 2 ), 'height' => absint( $img_src['height'] * 2 ) ) );
```

Comment:
I think the abs_int() should be applied to the variable before the multiplication, like:
`'width' => absint( $img_src['width'] ) * 2, 'height' => absint( $img_src['height'] ) * 2 `